config/settings.py

```diff
@@ -632,3 +650,37 @@
     raise ImproperlyConfigured(
         "SITE_BANNER_STYLE accepts only one of 'local', 'testing', 'production'."
     )
+
+# Feature Flags
+# -----------------------------------------------------------------------------
+# These flags are used to enable/disable features in AMY.
+# See https://cfpb.github.io/django-flags/ for more details.
+
+# ------------
+# The system for enabling or disabling feature flags by users themselves should only be
```

Comment:
Could you flesh out this documentation into an actual page in the developer docs? It would be helpful to explain the different ways a flag can be set through the UI/URL using these conditions, and how the conditions should be updated during the course of development.